features/admin_can_crud_article.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Admin can add an article
+  As an Admin
+  In order to add articles on the website
+  I would like to get CRUD access on my dashboard
+
+  Background:
+  Given The following Admin exists
+    | email               | password  |
+    | admin@example.com   | password  |
+  Given The following Article exists
+    | title               | content   |
+    | MyString            | MyText    |
+  Given I am on the Admin login page
+  Given I fill in "admin_user_email" with "admin@example.com"
+  Given I fill in "admin_user_password" with "password"
+  Given I click submit
+
+  Scenario: Admin successfully adds an article
+    When I click dashboard link Articles
+    And I click admin article page link "New Article"
+    Then I should be on "new_admin_article_path" dashboard page
+    And I fill in field "article_title" with "string"
```

Comment:
Try to avoid snake case on features, they need to be easily readable for not developers :)

Reply:
These are the values that active admin gives the field names, We need to use snake case in order for the field to be identified

Reply:
Oh, see my comment above.
Monkey Patch the String class perhaps?
```
class String
  def to_snake_case
    self.gsub(/::/, '/').
    gsub(/([A-Z]+)([A-Z][a-z])/,'\1_\2').
    gsub(/([a-z\d])([A-Z])/,'\1_\2').
    tr("-", "_").
    tr(" ", "_").
    downcase
  end
end
```

```
"Thomas Ochman".to_snake_case
=> "thomas_ochman"
```